scripts/linux_share_build.sh

```diff
@@ -98,19 +148,19 @@ function build_plugin_pkg(){
     cp -pPv ${PLUGIN_ABS_PATH}/DAGMC/lib/libmakeWatertight.so* .
     cp -pPv ${PLUGIN_ABS_PATH}/DAGMC/lib/libpyne_dagmc.so* .
     cp -pPv ${PLUGIN_ABS_PATH}/DAGMC/lib/libuwuw.so* .
-    cp -pPv /usr/lib/x86_64-linux-gnu/libhdf5_serial.so* .
+    cp -vL $HDF5_PATH/lib/libhdf5.so* .
     chmod 644 *
 
     # Set the RPATH to be the current directory for the DAGMC libraries
-    patchelf --set-rpath /opt/Trelis-${1::4}/bin/plugins/svalinn libMOAB.so
-    patchelf --set-rpath /opt/Trelis-${1::4}/bin/plugins/svalinn libdagmc.so
-    patchelf --set-rpath /opt/Trelis-${1::4}/bin/plugins/svalinn libmakeWatertight.so
-    patchelf --set-rpath /opt/Trelis-${1::4}/bin/plugins/svalinn libpyne_dagmc.so
-    patchelf --set-rpath /opt/Trelis-${1::4}/bin/plugins/svalinn libuwuw.so
+    patchelf --set-rpath ${TRELIS_PATH}/bin/plugins/svalinn libMOAB.so
+    patchelf --set-rpath ${TRELIS_PATH}/bin/plugins/svalinn libdagmc.so
+    patchelf --set-rpath ${TRELIS_PATH}/bin/plugins/svalinn libmakeWatertight.so
+    patchelf --set-rpath ${TRELIS_PATH}/bin/plugins/svalinn libpyne_dagmc.so
+    patchelf --set-rpath ${TRELIS_PATH}/bin/plugins/svalinn libuwuw.so
 
     # Create the Svalinn plugin tarball
     cd ..
     ln -sv svalinn/libsvalinn_plugin.so .
     cd ../..
-    tar --sort=name -czvf svalinn-plugin_linux_$1.tgz bin
+    tar --sort=name -czvf svalinn-plugin_linux_cubit_$1.tgz bin
```

Comment:
Just noticed that at some point we'll want to set the permissions of the tarball so that users other than `root` can access it (for Docker builds).

Reply:
?
this is not build as root.
or does the docker build make is as root by default ?

Reply:
Oh we run the docker as root, so it is create as root ?

Reply:
I just add a line to change permission to read/write for everyone.